src/styles/ForecastSummaries.css

```diff
@@ -0,0 +1,24 @@
+.forecast-summaries {
+  display: flex;
+  flex-flow: row wrap;
+  gap: 2rem;
+  margin-top: 3rem;
+}
+
+.forecast-summaries * {
+  flex-grow: 1;
+}
+
+.forecast-summaries .forecast-summary:not(:nth-of-type(-n + 2)) {
+  margin-top: 2rem;
+}
```

Comment:
This is a very specific selector... I think theres a much cleaner way of doing this. You can get the same effect by removing this rule completely, as well as its counterpart in the media query below.

As well as this, the use of the `.forecast-summaries * {}` to add the flex grow property to ALL child classes could be just as easily done by adding the `flex-grow: 1` property to `forecast-summary {}` in the forecast-summary file.